WORKSPACE

```diff
@@ -39,9 +39,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # 3) Check if envoy_build_config/extensions_build_config.bzl is up-to-date.
 # Try to match it with the one in source/extensions and comment out unneeded extensions.
 
-ENVOY_SHA1 = "b62dae29a5dd06b7f689899b26974d9567a98f0e"  # 2021-07-15: top of the tree after remote jwks retries
+ENVOY_SHA1 = "a2a1e3eed4214a38608ec223859fcfa8fb679b14"  # 2021-08-24: v1.19.1
```

Comment:
The comment could be easily confused with v1.19.1.  How about:
# Top of tree at 2021-08-24 after v1.19.1